torchgen/api/autograd.py

```diff
@@ -194,6 +195,34 @@ def create_view_copy_from_view_derivative(
             output_differentiability_conditions=self.output_differentiability_conditions,
         )
 
+    def create_functional_derivative(
```

Comment:
Most likely not needed either since https://github.com/pytorch/pytorch/blob/c25bdeea26f95d05281cc4309bc400a0eae0b8a4/torchgen/api/autograd.py#L314-L323 is there.

Reply:
oh I forgot to kill this now that I"m re-using `find_info()` to grab derivative entries for the generated ops